test/AngularSpec.js

```diff
@@ -1602,6 +1602,23 @@ describe('angular', function() {
     });
   });
 
+
+  describe('info', function() {
+    it('should return the additional info for the named module', function() {
+      angular.module('a', [], {some: 'thing'});
+      angular.module('b', ['dep'], {other: 'thang'}, function configFn() {});
+
+      expect(info('a')).toEqual({some: 'thing'});
```

Comment:
It is defined in the closure and on `angular` so we could make this test more clear.